assessments/2022-12-02-booleans/code.js

```diff
@@ -0,0 +1,41 @@
+//1
+const needHeavyCoat = (goingOutside, somewhereCold) => {
+  return goingOutside && somewhereCold
+}
+
+//2
+const needSunscreen = (goingToBeach, goingSkiing) => {
+  return goingToBeach || goingSkiing
+}
+
+//3
+const needMittens = (goingOutside, somewhereWarm) => {
+  return goingOutside && !somewhereWarm
+}
+
+//4 NOTE
+const isVenomous = (isStriped, hasBlueHead) => {
+  return !hasBlueHead
+}
+
+//5
+const okaySpeed = (speedLimit, carSpeed) => {
+  return (carSpeed > speedLimit - 11) && (carSpeed < speedLimit + 11)
```

Comment:
This would have been correct if you had used 11 rather than 10. But this would report someone going 19.5 in a 30 mile zone is okay since 19.5 > (30 - 11), i.e. 19.5 > 19. But that's 10.5 miles below the speed limit. I assume you were maybe trying to deal with folks going exactly 10 miles less but that would better handled like:

```suggestion
  return (carSpeed >= speedLimit - 10) && (carSpeed <= speedLimit + 10)
```